src/odemis/util/test/transform_test.py

```diff
@@ -329,6 +330,47 @@ def _angle_diff(x, y):
     return min(y - x, y - x + 2.0 * numpy.pi, y - x - 2.0 * numpy.pi, key=abs)
 
 
+class ToPhysicalSpaceKnownValues(unittest.TestCase):
+
+    def setUp(self):
+        self._ji = [(0, 0), (0, 4), (7, 0), (7, 4)]
+        self._xy = [(-2, 3.5), (2, 3.5), (-2, -3.5), (2, -3.5)]
+        self._shape = (8, 5)
+
+    def test_to_physical_space_known_values(self):
+        """
+        to_physical_space should return known result with known input.
+
+        """
+        for ji, xy in zip(self._ji, self._xy):
+            res = to_physical_space(ji, self._shape)
+            numpy.testing.assert_array_almost_equal(xy, res)
```

Comment:
Why did you not use `assert_array_equal` ?

Reply:
We're comparing floats here. This is to prevent rounding errors to become errors. For the purpose of this unit test 0.199999999 is equal to 0.2